src/storage/engine/rocksdb.rs

```diff
@@ -54,6 +54,20 @@ impl Engine for EngineRocksdb {
         Ok(pair)
     }
 
+    fn reverse_seek(&self, _: &Context, key: &Key) -> Result<Option<KvPair>> {
+        trace!("EngineRocksdb: seek {:?}", key);
+        let mut iter = self.db.iterator(IteratorMode::From(key.raw(), Direction::Reverse));
+        // iter will be positioned at `key` or the kv pair after it. If no such key exists, we need
+        // locate it to the end.
+        if !iter.valid() {
```

Comment:
@BusyJay I will add it after https://github.com/ngaut/rust-rocksdb/pull/7 be merged.